commands/issue/create/issue_create.go

```diff
@@ -163,14 +161,14 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 			if opts.MileStone != -1 {
 				issueCreateOpts.MilestoneID = gitlab.Int(opts.MileStone)
 			}
-			if opts.Assignees != "" {
-				arrIds := strings.Split(strings.Trim(opts.Assignees, "[] "), ",")
-				var assigneeIDs []int
-
-				for _, id := range arrIds {
-					assigneeIDs = append(assigneeIDs, utils.StringToInt(id))
+			if len(opts.Assignees) > 0 {
+				users, err := api.UsersByNames(apiClient, opts.Assignees)
+				if err != nil {
+					return err
+				}
+				for _, user := range users {
+					issueCreateOpts.AssigneeIDs = append(issueCreateOpts.AssigneeIDs, user.ID)
 				}
-				issueCreateOpts.AssigneeIDs = assigneeIDs
 			}
```

Comment:
Looks like this will be repeated for several commands. Maybe we could refactor and create it as a function in ./commands/cmdutils/cmdutils.go. 
What do you think?

Reply:
Yeah, I thought of it but didnt know if that is really needed.
Gonna extract this to some general userID extracting function.